Makefile

```diff
@@ -58,6 +58,7 @@ TEST_REQUIRES_ROOT_PACKAGES=$(filter \
 # Project binaries.
 COMMANDS=ctr containerd containerd-stress containerd-release
 
+BUILDTAGS ?= seccomp apparmor
```

Comment:
Please add a comment about this to Makefile?

Reply:
Done. And also updated BUILDING.md